src/Lock - Closed - Fill.svg

```diff
@@ -1,3 +1,3 @@
 <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This file rename is going to be a breaking change if we don't add an additional file with the old name.

Reply:
I assume this icon changed names when we added the "Outline" version. I don't mind adding back in the old name, but what would be a good way to avoid this in the future? Do we just have duplicate icons with old names around or in an ideal world do we have any better options?

Reply:
Because the SVGs are being consumed directly, the only way to avoid a breaking change is to have duplicate SVGs with different names.

Reply:
However, if we were to ship a JavaScript module with named exports, we could just define aliases instead of duplicating definitions. (Not sure if this would be consumable by stakeholders, though.)

Reply:
Ok. Added the old ones back in too. I was trying to think of something like making a list of "aliases" so we could generate the duplicates based on that, but maybe we'll just have to be careful.